contracts/governance/Staking/Staking.sol

```diff
@@ -774,4 +939,13 @@ contract Staking is
                 block.number;
         require(notSameBlock, "S20"); //S20 : "cannot be mined in the same block as last stake"
     }
+
+    /**
+     * @notice Max iteration for vesting withdrawal to prevent out of gas issue.
+     *
+     * @return max iteration value.
+     */
+    function getMaxVestingWithdrawIterations() public pure returns (uint256) {
```

Comment:
pls make it storage var settable by onlyAuthorized addresses
else we would have to go via SIP to change this value which is an overkill